.chloggen/jackgopack4-add-converters-component-command.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: otelcol
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Converters are now available in the `components` command. 
+
+# One or more tracking issues or pull requests related to the change
+issues: [11900, 12385]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: buildinfo in components command is now reported as "build_info" for legibility.
```

Comment:
Let's keep the PRs focused on a single purpose. AFAICT this is independent from the other changes, so I would like to drop it or move it to a separate PR